.dev/tests/cypress/helpers.js

```diff
@@ -46,9 +46,8 @@ export function loginToSite() {
 				cy.get( '#user_pass' ).type( Cypress.env( 'wpPassword' ) );
 				cy.get( '#wp-submit' ).click();
 			}
-		} );
-
-	cy.get( '.block-editor-page' ).should( 'exist' );
+		} )
+		.get( '.wp-block-post-title' ).should( 'exist' );
```

Comment:
Smart. I'm assuming this will make Cypress wait until the editor is actually loaded rather than the DOM surrounding the React app.

Reply:
Yes, it makes it wait a little longer :)